addons/strimzi/duck/v1beta2/duck_types.go

```diff
@@ -40,6 +40,12 @@ const (
 type KafkaTopic struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Status            KafkaTopicStatus `json:"status,omitempty"`
+}
+
+// KafkaTopicStatus is the duck of a KafkaTopic status.
+type KafkaTopicStatus struct {
+	TopicName string `json:"topicName,omitempty"`
```

Comment:
Not sure if we need to add a `status` subresource permission https://github.com/apache/camel-k/blob/main/pkg/resources/config/rbac/descoped/operator-cluster-role-strimzi.yaml

Reply:
Good point. I'll have a look, thanks.

Reply:
Should be fixed now.